ggml/src/CMakeLists.txt

```diff
@@ -566,6 +566,17 @@ if (GGML_SYCL)
         list(APPEND GGML_EXTRA_LIBS_PRIVATE DNNL::dnnl)
     endif()
 
+    set(oneCCL_DIR "/opt/intel/oneapi/ccl/latest/lib/cmake/oneCCL")
```

Comment:
The real oneapi path is not always in `/opt/intel/oneapi/`.
Please use `ENV{ONEAPI_ROOT}` which is mandatory env variable in cmakefile.